src/Mod/CAM/Path/Post/Command.py

```diff
@@ -137,6 +137,7 @@ def _write_file(self, filename, gcode, policy):
                 Path.Log.debug(filename)
                 with open(filename, "w") as f:
                     f.write(gcode)
+            else: return
```

Comment:
Here, the dialog "cancel" action was ignored, resulting in the action being performed even if the dialog was cancelled